.vscode/settings.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
I think this file was committed by mistake. You probably need to add `.vscode` in `.gitignore`.